nucleo.cwl

```diff
@@ -0,0 +1,810 @@
+          - 0
+        source:
+          - fgbio_call_duplex_consensus_reads_min_reads
+      - id: reference_sequence
+        source: reference_sequence
+      - id: validation_stringency
+        default: LENIENT
+        source: validation_stringency
+      - id: gatk_sam_to_fastq_output_name_R2
+        source: BC_gatk_sam_to_fastq_output_name_R2
+      - id: gatk_sam_to_fastq_output_name_R1
+        source: BC_gatk_sam_to_fastq_output_name_R1
+      - id: bwa_mem_Y
+        default: true
+        source: bwa_mem_Y
+      - id: bwa_mem_T
+        default: 30
+        source: bwa_mem_T
+      - id: sort_order
+        default: coordinate
+      - id: picard_addRG_read_group_sequencing_platform
+        default: ILLUMINA
+        source: platform
+      - id: picard_addRG_read_group_sequencing_center
+        default: MSKCC
+        source: sequencing-center
+      - id: picard_addRG_read_group_run_date
+        source: run-date
+      - id: picard_addRG_read_group_platform_unit
+        source: platform-unit
+      - id: picard_addRG_read_group_library
+        source: library
+      - id: picard_addRG_read_group_identifier
+        source: read-group-id
+      - id: picard_addRG_output_file_name
+        source: BC_picard_addRG_output_file_name
+      - id: bwa_mem_output
+        source: BC_bwa_mem_output
+      - id: create_bam_index
+        default: true
+        source: create_bam_index
+      - id: bwa_mem_K
+        default: 1000000
+      - id: abra2_window_size
+        default: '800,700'
+        source: abra2_window_size
+      - id: abra2_soft_clip_contig
+        default: '100,30,80,15'
+        source: abra2_soft_clip_contig
+      - id: abra2_scoring_gap_alignments
+        default: '8,32,48,1'
+        source: abra2_scoring_gap_alignments
+      - id: picard_fixmate_information_output_file_name
+        source: BC_picard_fixmate_information_output_file_name
+      - id: abra2_output_bams
+        source:
+          - BC_abra2_output_bams
+      - id: bedtools_genomecov_option_bedgraph
+        default: true
+        source: bedtools_genomecov_option_bedgraph
+      - id: abra2_no_edge_complex_indel
+        default: true
+        source: abra2_no_edge_complex_indel
+      - id: abra2_maximum_mixmatch_rate
+        default: 0.1
+        source: abra2_maximum_mixmatch_rate
+      - id: abra2_maximum_average_depth
+        default: 1000
+        source: abra2_maximum_average_depth
+      - id: bedtools_merge_distance_between_features
+        default: 10
+        source: bedtools_merge_distance_between_features
+      - id: abra2_contig_anchor
+        default: '10,1'
+        source: abra2_contig_anchor
+      - id: abra2_consensus_sequence
+        default: true
+        source: abra2_consensus_sequence
+      - id: gatk_merge_bam_alignment_output_file_name
+        source: BC_gatk_merge_bam_alignment_output_file_name
+      - id: fgbio_filter_consensus_read_reverse_per_base_tags_simplex_duplex
+        default: true
```

Comment:
I'm not aware of this parameter and why it needs to be `true` but if it is necessary for the workflow to run I'm ok with it

Reply:
> I'm not aware of this parameter and why it needs to be `true` but if it is necessary for the workflow to run I'm ok with it

I think this is what @kanika-arora  had in her commands, I remember she gave an explanation but cannot recall, @kanika-arora  do you recall.